README.md

```diff
@@ -173,14 +172,14 @@ Now it's time to create a Solidity smart contract.
 
    - And that's it! Your code should look like [this](https://github.com/LearnWeb3DAO/BasicFrontEndTutorial/blob/master/contracts/mood.sol)
 
-6. Deploy the contract on the Ropsten Testnet.
-   - Make sure your Metamask is connected to the Ropsten Testnet.
+6. Deploy the contract on the Goerli Testnet.
+   - Make sure your Metamask is connected to the Goerli Testnet.
    - Make sure you select the right compiler version to match the solidity contract. (In the compile tab)
    - Compile the code using the "Solidity Compiler" tab. _Note that it may take a moment to load the compiler_
    - Deploy the contract under the "Deploy and Run Transactions" tab
    - Under the Deployed Contracts section, you can test out your functions on the Remix Run tab to make sure your contract works as expected!
 
-**_Be sure to deploy on Ropsten via Remix under the `Injected Web3` environment and confirm the deployment transaction in Metamask_**
+**_Be sure to deploy on Goerli via Remix under the `Injected Web3` environment and confirm the deployment transaction in Metamask_**
```

Comment:
I think it's no longer 'Injected Web3' on Remix

Reply:
![image](https://user-images.githubusercontent.com/71306738/191206197-b469fa97-5a92-456c-ae38-25a2f7c1c054.png)

kindly check here , it's injected